rawfile/utils/rawfile.py

```diff
@@ -49,9 +51,14 @@ class AccessType(Enum):
 
 def path_stats(path):
     fs_stat = os.statvfs(path)
+
+    total = CONFIG.get("capacity_override", 0) or (fs_stat.f_frsize * fs_stat.f_blocks)
```

Comment:
Not sure if we can do this here, path_stats may be called for a volume rather than the data_dir for volumes.
In such case we should report fs_avail of the volume itself?

Reply:
Also should truncate this to the actual available capacity on the node?